api/controllers/subscription.js

```diff
@@ -119,91 +119,80 @@ async function syncSubscriptions(req, res) {
   await gapiAuth();
   const params = { packageName: 'com.unicef.cboard' };
   let paypalPlans = [];
+  
   // get PayPal plans 
   try {
     paypalPlans = await paypal.listPlans();
   } catch (err) {
     console.log(err.message);
   }
+  
   try {
     // get subscriptions from Google API
     const remoteData = await playConsole.monetization.subscriptions.list(params);
     const remoteSubscrs = remoteData.data.subscriptions;
-    //loop remote subscriptions
-    remoteSubscrs.forEach(subscription => {
-      //console.log(subscription);
-      const subscriptionId = subscription.productId;
-      Subscription.findOne({ subscriptionId: subscriptionId }, async function (err, subscr) {
-        if (err) {
-          return res.status(500).json({
-            message: 'Error getting subscription.',
-            error: err.message
-          });
-        }
+    
+    // loop remote subscriptions
+    for (const subscription of remoteSubscrs) {
+      try {
+        const subscriptionId = subscription.productId;
+        const subscr = await Subscription.findOne({ subscriptionId: subscriptionId });
```

Comment:
Now we are trying to minimize the use of abreviations in our code to have more clear code. The old code was not reflecting that hahaha. 
What do you think on rename `subscr` to `userSubcription` or   something similar?